notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT:</p><p>- Intuatively (and imprecisely) -&gt;&gt; Just say Intuitively</p><p>- I'm not sure we can say "the connection between u and v", at least I have never heard it. I think it might be called "connection" because it can "connect"/relate tangent spaces. --&gt; I would remove "the connection between&nbsp;u&nbsp;and&nbsp;v, or"</p><p> </p><p>Always refer to Figures (it's good habit for research papers), e.g. say something like "see figure below."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done :D</p>